Source/JavaScriptCore/jit/JITPropertyAccess.cpp

```diff
@@ -1535,6 +1569,18 @@ void JIT::emit_op_put_to_scope(const JSInstruction* currentInstruction)
 
     ResolveType profiledResolveType = bytecode.metadata(m_profiledCodeBlock).m_getPutInfo.resolveType();
 
+    constexpr GPRReg metadataGPR = regT5;
+    using Metadata = OpPutToScope::Metadata;
+    static_assert(!(Metadata::offsetOfGetPutInfo() % 8));
+    static_assert(!(Metadata::offsetOfStructure() % 8));
+    static_assert(!(Metadata::offsetOfOperand() % 8));
+    static_assert(!(Metadata::offsetOfWatchpointSet() % 8));
```

Comment:
Ditto: let's use a `constexpr size_t minAlignment = 8;` here instead of these literal 8s.

Reply:
Added in latest push - actually based on `alignof(void*)` since these are pointers and not 64-bit values.